features/pause_and_resume.feature

```diff
@@ -54,55 +54,6 @@ Feature: Pause and Resume User Journey
     When they give their consent
     Then they should be successfully verified
 
-  Scenario: User pauses and resumes from an IDP e-mail link, on same device, and resumes registration
```

Comment:
covered in here: https://github.com/alphagov/verify-frontend/blob/master/spec/features/user_visits_paused_page_spec.rb